Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,56 @@
+//1. Declare a variable on each type using two options: plain creation, creation with constructor.
+const stringVar = 'Product Owner';
+const booleanVar = true;
+const nullVar = null;
+let undefinedVar; 
+const numberVar = new Number(32);
+const objectVar = new Object('prior experience');
```

Comment:
You can also check this comment to get some information about declaring numbers/strings/booleans using `new` operator:
https://stackoverflow.com/a/4719334